docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -0,0 +1,373 @@
+      - `None`, which means that you don't consider the movement direction and
+         calculate the velocity by the real distance. This is the default value.
+      - Any real number from `0` to `360` which represents the angular information
+        of the direction in the coordination system.
+        Note that the axis can be represented either by `X+`, `Y+`, `X-`, `Y-` or
+        by 0, 90, 180, 270.
+      - `SeeTraj`. For complex trajectories with several times of direction change,
+         you can indicate the detailed direction using the
+         angular information in the trajectory file
+         (By adding a new column in `.txt` file or adding a new variable in `.xml`
+         file with the indicator `VD`).
+
+  - `ignore_backward_movement` indicates whether you want to ignore the movement
+     opposite to the direction from `set_movement_direction`.
+     The default value is `false`.
+
+
+## Methods
+
+  Indicates the parameters related to each measurement method.
+  Six different methods `method_A` to `method_D` and `Method_I` to `Method_J` are integrated in the current
+  version of `JPSreport` and can be chosen for the analysis. The five methods `method_A` to `method_D` and `method_J` are used to analyze the steady state.
+
+| Method | measurement area | output data |
+| :---:  |     :---:        | :---:       |
+| *A* | ![Method A]({{ site.baseurl }}/images/jpsreport_Method_A.png) |  $$\langle v \rangle_{\Delta t}$$ and  $$\langle J \rangle_{\Delta t}$$ |
+| *B* | ![Method B]({{ site.baseurl }}/images/jpsreport_Method_B.png) | $$\langle v \rangle_i$$ and $$\langle \rho \rangle_i$$ |
+| *C* | ![Method C]({{ site.baseurl }}/images/jpsreport_Method_C.png) | $$\langle v \rangle_{\Delta x}$$ and $$\langle \rho \rangle_{\Delta x}$$ |
+| *D* | ![Method D]({{ site.baseurl }}/images/jpsreport_Method_D.png) | $$\langle v \rangle_v$$ and $$\langle \rho \rangle_v$$ |
+| *J* | ![Method J]({{ site.baseurl }}/images/jpsreport_Method_D.png) | $$\langle v \rangle_v$$ and $$\langle \rho \rangle_v$$ |
+
+  Further information relating to each method can be found
+  in [Pedestrian fundamental diagrams: Comparative analysis of experiments in different geometries](http://hdl.handle.net/2128/4898).
+
+`Method_I` can be used for time-series analysis of individual data.
+
+
+### Method A
+For definition see [Method A](jpsreport_method_A.html). Method A is used to analyze the steady state.
+
+```xml
+  <method_A enabled="true">
+    <measurement_area id="2" frame_interval="100"/>
+    <measurement_area id="3" frame_interval="100"/>
+  </method_A>
+```
+
+Possible parameters are:
+- `id` specifies the location of the reference line.
+several measurement areas can be set in one inifile with different id-numbers.
+
+- `frame_interval` specifies the size of time interval (in *frame*)
+   for calculating flow rate.
+
+- up to version 0.8.5: `plot_time_series` specifies whether output the ($$N-t$$)-Diagram.
```

Comment:
The version number must be corrected.

Reply:
Here again, I don't know the correct number. :-(

Reply:
here again, good question. After the union of jpscore and jpsreport, @zeroset and you (@chraibi) should clarify this. JPScore is currently at 0.8.4. JPSreport was already at 0.8.5.

Reply:
see [suggestion](https://github.com/JuPedSim/jpscore/pull/626#discussion_r360882255)